/* Creator:       Michelle Konzack                */
/* E-Mail:        <linux4michelle@freenet.de>     */
/*                                                */
/* Creation-Date: Sat, 12 Mar 2005 16:59:01 +0100 */


--- syntax/sh.syntax	Sat Mar 12 16:54:56 2005
+++ syntax/sh.syntax	Sat Mar 12 16:57:47 2005
@@ -41,7 +41,9 @@ wholechars abcdefghijklmnopqrstuvwxyzABC
     keyword whole bg yellow
     keyword whole break yellow
     keyword whole case yellow
+    keyword whole clear yellow
     keyword whole continue yellow
+    keyword whole declare yellow
     keyword whole done yellow
     keyword whole do yellow
     keyword whole elif yellow
@@ -56,11 +58,13 @@ wholechars abcdefghijklmnopqrstuvwxyzABC
     keyword whole for yellow
     keyword whole if yellow
     keyword whole in yellow
+    keyword whole let yellow
     keyword whole read yellow
     keyword whole return yellow
     keyword whole select yellow
     keyword whole set yellow
     keyword whole shift yellow
+    keyword whole source yellow
     keyword whole then yellow
     keyword whole trap yellow
     keyword whole umask yellow
@@ -69,6 +73,8 @@ wholechars abcdefghijklmnopqrstuvwxyzABC
     keyword whole wait yellow
     keyword whole while yellow
 
+    keyword whole apt-get cyan
+    keyword whole ar cyan
     keyword whole arch cyan
     keyword whole ash cyan
     keyword whole awk cyan
@@ -77,90 +83,204 @@ wholechars abcdefghijklmnopqrstuvwxyzABC
     keyword whole bg_backup cyan
     keyword whole bg_restore cyan
     keyword whole bsh cyan
+    keyword whole bzip cyan
+    keyword whole bzip2 cyan
+    keyword whole cam cyan
     keyword whole cat cyan
     keyword whole cd cyan
+    keyword whole cdda2wav cyan
     keyword whole chgrp cyan
     keyword whole chmod cyan
     keyword whole chown cyan
+    keyword whole cmp cyan
+    keyword whole col cyan
+    keyword whole compress cyan
     keyword whole cp cyan
     keyword whole cpio cyan
+    keyword whole cpp cyan
     keyword whole csh cyan
+    keyword whole curl cyan
+    keyword whole cut cyan
     keyword whole date cyan
     keyword whole dd cyan
     keyword whole df cyan
+    keyword whole dialog cyan
+    keyword whole diff cyan
+    keyword whole dirname cyan
     keyword whole dmesg cyan
     keyword whole dnsdomainname cyan
     keyword whole doexec cyan
     keyword whole domainname cyan
+    keyword whole dpkg cyan
+    keyword whole dpkg-buildpackage cyan
+    keyword whole dpkg-scanpackages cyan
+    keyword whole dpkg-scansources cyan
     keyword whole echo cyan
     keyword whole ed cyan
+    keyword whole editor cyan
     keyword whole egrep cyan
     keyword whole ex cyan
+    keyword whole fakeroot cyan
     keyword whole false cyan
+    keyword whole fdformat cyan
+    keyword whole fetchmail cyan
     keyword whole fgrep cyan
     keyword whole find cyan
+    keyword whole formail cyan
+    keyword whole free cyan
+    keyword whole freeze cyan
     keyword whole fsconf cyan
     keyword whole gawk cyan
+    keyword whole gdb cyan
+    keyword whole gcc cyan
     keyword whole grep cyan
     keyword whole gunzip cyan
     keyword whole gzip cyan
+    keyword whole ha cyan
     keyword whole hostname cyan
     keyword whole igawk cyan
+    keyword whole insmod cyan
     keyword whole ipcalc cyan
+    keyword whole iptables cyan
     keyword whole kill cyan
     keyword whole ksh cyan
+    keyword whole lame cyan
+    keyword whole less cyan
+    keyword whole lharc cyan
+    keyword whole lilo cyan
+    keyword whole linux_logo cyan
     keyword whole linuxconf cyan
     keyword whole ln cyan
+    keyword whole locale cyan
+    keyword whole logger cyan
     keyword whole login cyan
     keyword whole lpdconf cyan
     keyword whole ls cyan
+    keyword whole lynx cyan
     keyword whole mail cyan
+    keyword whole man cyan
+    keyword whole mc cyan
+    keyword whole mcedit cyan
+    keyword whole mcview cyan
+    keyword whole mimedecode cyan
     keyword whole mkdir cyan
+    keyword whole mkdirhier cyan
+    keyword whole mkfs.ext2 cyan
+    keyword whole mkfs.ext3 cyan
+    keyword whole mkfs.minix cyan
+    keyword whole mkfs.msdos cyan
+    keyword whole mke2fs cyan
+    keyword whole mkdosfs cyan
+    keyword whole mkinitrd cyan
     keyword whole mknod cyan
     keyword whole mktemp cyan
+    keyword whole modprobe cyan
     keyword whole more cyan
     keyword whole mount cyan
+    keyword whole mozilla cyan
+    keyword whole mp3info cyan
+    keyword whole munpack cyan
+    keyword whole msgfmt cyan
     keyword whole mt cyan
+    keyword whole mutt cyan
     keyword whole mv cyan
     keyword whole netconf cyan
     keyword whole netstat cyan
     keyword whole nice cyan
     keyword whole nisdomainname cyan
+    keyword whole nm cyan
+    keyword whole patch cyan
+    keyword whole pinfo cyan
     keyword whole ping cyan
+    keyword whole procmail cyan
     keyword whole ps cyan
     keyword whole pwd cyan
+    keyword whole rar cyan
     keyword whole red cyan
     keyword whole remadmin cyan
+    keyword whole rename cyan
     keyword whole rm cyan
     keyword whole rmdir cyan
+    keyword whole rmmod cyan
+    keyword whole rplay cyan
     keyword whole rpm cyan
+    keyword whole rpm2cpio cyan
     keyword whole sed cyan
     keyword whole set cyan
     keyword whole setserial cyan
     keyword whole sh cyan
     keyword whole sleep cyan
     keyword whole sort cyan
+    keyword whole sa-learn cyan
+    keyword whole spamassassin cyan
+    keyword whole spamc cyan
+    keyword whole spamd cyan
+    keyword whole ssmtp cyan
+    keyword whole strace cyan
     keyword whole stty cyan
     keyword whole su cyan
     keyword whole sync cyan
     keyword whole taper cyan
     keyword whole tar cyan
+    keyword whole tcpdump cyan
     keyword whole tcsh cyan
     keyword whole test cyan
+    keyword whole tempfile cyan
     keyword whole time cyan
+    keyword whole tnef cyan
     keyword whole touch cyan
+    keyword whole tr cyan
     keyword whole true cyan
+    keyword whole tune2fs cyan
     keyword whole umount cyan
     keyword whole uname cyan
+    keyword whole unarj cyan
+    keyword whole uniq cyan
+    keyword whole unzip cyan
+    keyword whole uptime cyan
     keyword whole userconf cyan
     keyword whole usleep cyan
     keyword whole vi cyan
     keyword whole view cyan
     keyword whole vim cyan
+    keyword whole wc cyan
+    keyword whole wget cyan
+    keyword whole whiptail cyan
+    keyword whole wvWare cyan
     keyword whole xconf cyan
+    keyword whole xgettext cyan
+    keyword whole xmessage cyan
+    keyword whole xmodmap cyan
+    keyword whole xterm cyan
     keyword whole ypdomainname cyan
     keyword whole zcat cyan
+    keyword whole zgv cyan
+    keyword whole zip cyan
+    keyword whole zoo cyan
     keyword whole zsh cyan
+    keyword whole Xdialog cyan
+
+    keyword whole gpg     red
+    keyword whole md5sum  red
+    keyword whole openssl red
+    keyword whole ssh     red
+
+    keyword whole TEXTDOMAINDIR magenta
+    keyword whole TEXTDOMAIN    magenta
+    keyword whole VERSION       magenta
+
+    keyword whole jpegtopnm cyan
+    keyword whole pnmscale  cyan
+    keyword whole ppmtoxpm  cyan
+
+    keyword whole /dev/audio  brightblue
+    keyword whole /dev/dsp    brightblue
+    keyword whole /dev/null   brightblue
+    keyword whole /dev/mixed  brightblue
+    keyword whole /dev/stdin  brightblue
+    keyword whole /dev/stdout brightblue
+    keyword whole /dev/stderr brightblue
+    keyword whole /dev/zero   brightblue
 
 wholechars abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_
 
@@ -224,7 +344,9 @@ wholechars abcdefghijklmnopqrstuvwxyzABC
 
     keyword whole break yellow
     keyword whole case yellow
+    keyword whole clear yellow
     keyword whole continue yellow
+    keyword whole declare yellow
     keyword whole done yellow
     keyword whole do yellow
     keyword whole elif yellow
@@ -238,11 +360,15 @@ wholechars abcdefghijklmnopqrstuvwxyzABC
     keyword whole return yellow
     keyword whole select yellow
     keyword whole shift yellow
+    keyword whole source yellow
     keyword whole then yellow
     keyword whole trap yellow
     keyword whole until yellow
+    keyword whole wait yellow
     keyword whole while yellow
 
+    keyword whole apt-get cyan
+    keyword whole ar cyan
     keyword whole arch cyan
     keyword whole ash cyan
     keyword whole awk cyan
@@ -252,87 +378,194 @@ wholechars abcdefghijklmnopqrstuvwxyzABC
     keyword whole bg_backup cyan
     keyword whole bg_restore cyan
     keyword whole bsh cyan
+    keyword whole bzip cyan
+    keyword whole bzip2 cyan
+    keyword whole cam cyan
     keyword whole cat cyan
     keyword whole cd cyan
+    keyword whole cdda2wav cyan
     keyword whole chgrp cyan
     keyword whole chmod cyan
     keyword whole chown cyan
+    keyword whole cmp cyan
+    keyword whole col cyan
+    keyword whole compress cyan
     keyword whole cp cyan
     keyword whole cpio cyan
+    keyword whole cpp cyan    
     keyword whole csh cyan
+    keyword whole curl cyan
+    keyword whole cut cyan
     keyword whole date cyan
     keyword whole dd cyan
     keyword whole df cyan
+    keyword whole dialog cyan
+    keyword whole diff cyan
+    keyword whole dirname cyan
     keyword whole dmesg cyan
     keyword whole dnsdomainname cyan
     keyword whole doexec cyan
     keyword whole domainname cyan
+    keyword whole dpkg cyan
+    keyword whole dpkg-buildpackage cyan
+    keyword whole dpkg-scanpackages cyan
+    keyword whole dpkg-scansources cyan
     keyword whole echo cyan
     keyword whole ed cyan
+    keyword whole editor cyan
     keyword whole egrep cyan
     keyword whole ex cyan
+    keyword whole fakeroot cyan
     keyword whole false cyan
+    keyword whole fdformat cyan
+    keyword whole fetchmail cyan
     keyword whole fgrep cyan
     keyword whole find cyan
+    keyword whole free cyan
+    keyword whole freeze cyan
     keyword whole fsconf cyan
     keyword whole gawk cyan
+    keyword whole gdb cyan
+    keyword whole gcc cyan
     keyword whole grep cyan
     keyword whole gunzip cyan
     keyword whole gzip cyan
+    keyword whole ha cyan
     keyword whole hostname cyan
     keyword whole igawk cyan
+    keyword whole insmod cyan
+    keyword whole iptables cyan
     keyword whole ipcalc cyan
     keyword whole kill cyan
     keyword whole ksh cyan
+    keyword whole lame cyan
+    keyword whole less cyan
+    keyword whole lharc cyan
+    keyword whole lilo cyan
+    keyword whole linux_logo cyan
     keyword whole linuxconf cyan
     keyword whole ln cyan
+    keyword whole locale cyan
+    keyword whole logger cyan
     keyword whole login cyan
     keyword whole lpdconf cyan
     keyword whole ls cyan
+    keyword whole lynx cyan
     keyword whole mail cyan
+    keyword whole man cyan
+    keyword whole mc cyan
+    keyword whole mcedit cyan
+    keyword whole mcview cyan
+    keyword whole mimedecode cyan
     keyword whole mkdir cyan
+    keyword whole mkdirhier cyan
+    keyword whole mkfs.ext2 cyan
+    keyword whole mkfs.ext3 cyan
+    keyword whole mkfs.minix cyan
+    keyword whole mkfs.msdos cyan
+    keyword whole mke2fs cyan
+    keyword whole mkdosfs cyan
+    keyword whole mkinitrd cyan
     keyword whole mknod cyan
     keyword whole mktemp cyan
+    keyword whole modprobe cyan
     keyword whole more cyan
     keyword whole mount cyan
+    keyword whole mozilla cyan
+    keyword whole mp3info cyan
+    keyword whole msgfmt cyan
     keyword whole mt cyan
+    keyword whole mutt cyan
     keyword whole mv cyan
     keyword whole netconf cyan
     keyword whole netstat cyan
     keyword whole nice cyan
     keyword whole nisdomainname cyan
+    keyword whole nm cyan
+    keyword whole patch cyan
+    keyword whole pinfo cyan
     keyword whole ping cyan
     keyword whole ps cyan
     keyword whole pwd cyan
+    keyword whole rar cyan
     keyword whole red cyan
     keyword whole remadmin cyan
+    keyword whole rename cyan
     keyword whole rm cyan
     keyword whole rmdir cyan
+    keyword whole rmmod cyan
+    keyword whole rplay cyan
     keyword whole rpm cyan
+    keyword whole rpm2cpio cyan
     keyword whole sed cyan
     keyword whole set cyan
     keyword whole setserial cyan
     keyword whole sh cyan
     keyword whole sleep cyan
     keyword whole sort cyan
+    keyword whole sa-learn cyan
+    keyword whole spamassassin cyan
+    keyword whole spamc cyan
+    keyword whole spamd cyan
+    keyword whole ssmtp cyan
+    keyword whole strace cyan
     keyword whole stty cyan
     keyword whole su cyan
     keyword whole sync cyan
     keyword whole taper cyan
     keyword whole tar cyan
+    keyword whole tcpdump cyan
     keyword whole tcsh cyan
+    keyword whole tempfile cyan
     keyword whole test cyan
     keyword whole time cyan
+    keyword whole tnef cyan
     keyword whole touch cyan
+    keyword whole tr cyan
     keyword whole true cyan
+    keyword whole tune2fs cyan
     keyword whole umount cyan
     keyword whole uname cyan
+    keyword whole unarj cyan
+    keyword whole uniq cyan
+    keyword whole unzip cyan
+    keyword whole uptime cyan
     keyword whole userconf cyan
     keyword whole usleep cyan
     keyword whole vi cyan
     keyword whole view cyan
     keyword whole vim cyan
+    keyword whole wc cyan
+    keyword whole wget cyan
+    keyword whole whiptail cyan
+    keyword whole wvWare cyan
     keyword whole xconf cyan
+    keyword whole xgettext cyan
+    keyword whole xmessage cyan
+    keyword whole xmodmap cyan
+    keyword whole xterm cyan
     keyword whole ypdomainname cyan
     keyword whole zcat cyan
+    keyword whole zgv cyan
+    keyword whole zoo cyan
+    keyword whole zip cyan
     keyword whole zsh cyan
+    keyword whole Xdialog cyan
+
+    keyword whole gpg     red
+    keyword whole md5sum  red
+    keyword whole openssl red
+    keyword whole ssh     red
+
+    keyword whole TEXTDOMAINDIR magenta
+    keyword whole TEXTDOMAIN    magenta
+    keyword whole VERSION       magenta
+
+    keyword whole /dev/audio  brightblue
+    keyword whole /dev/dsp    brightblue
+    keyword whole /dev/null   brightblue
+    keyword whole /dev/mixed  brightblue
+    keyword whole /dev/stdin  brightblue
+    keyword whole /dev/stdout brightblue
+    keyword whole /dev/stderr brightblue
+    keyword whole /dev/zero   brightblue
